### PR TITLE
Tests for #4386

### PR DESCRIPTION
**Pull request by vtols**

### CODE REVIEW

File: src/test/java/com/axibase/tsd/api/method/sql/syntax/SqlSyntaxLocalFormatTest.java [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+     */
+    @Test(
+            description = "The test asserts that dates with different digits " +
+                    "after 3rd fractional place are indistinguishable in local " +
+                    "date format"
+    )
+    public void testLocalDateFractionalPartTruncationTo3() {
+        String sqlQuery = String.format(
+                "SELECT datetime " +
+                        "FROM '%1$s' " +
```
Comment (alexey-kravtsov) [inline src/test/java/com/axibase/tsd/api/method/sql/syntax/SqlSyntaxLocalFormatTest.java]: In future, you don't have to explicitly add index if parameters are passed in the same order. So %1$s, %2$s may be replaced with %s, %s

File: src/test/java/com/axibase/tsd/api/method/sql/syntax/SqlSyntaxLocalFormatTest.java [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+                    "three decimal places aren't supported for dates in local " +
+                    "date format"
+    )
+    public void testLocalDateLessThanThreeFractionalDigits() {
+        String beginDate = stringTruncate(TestUtil.formatAsLocalTime("2017-01-01T00:00:00.998Z"), 1);
+        String endDate = stringTruncate(TestUtil.formatAsLocalTime("2017-01-01T00:00:01.000Z"), 2);
+
+        String sqlQuery = String.format(
+                "SELECT datetime " +
+                        "FROM '%1$s' " +
```
Comment (raipc) [inline src/test/java/com/axibase/tsd/api/method/sql/syntax/SqlSyntaxLocalFormatTest.java]: Please escape metric names with double quotes

File: src/test/java/com/axibase/tsd/api/method/sql/syntax/SqlSyntaxLocalFormatTest.java [hunk 3]
```diff
@@ -0,0 +1,206 @@
...
+     */
+    private String stringTruncate(String value, int tailLength) {
+        return value.substring(0, value.length() - tailLength);
+    }
+
+    @BeforeClass
+    public static void prepareData() throws Exception {
+        Series series = new Series(Mocks.entity(), METRIC_NAME);
+        series.addSamples(
+                new Sample("2017-01-01T00:00:00.997Z", 1),
```
Comment (rmakulov) [inline src/test/java/com/axibase/tsd/api/method/sql/syntax/SqlSyntaxLocalFormatTest.java]: #371

**Comment by rmakulov** [comment]
Please, resolve conflicts.

**Pull request merged**